### PR TITLE
Including Bxml support for python SDK

### PR DESCRIPTION
**Pull request by dpereira-daitan**

### CODE REVIEW

File: bandwidth_sdk/xml.py [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+        result = []
+        for verb in self.verbs:
+            result.append(verb.to_xml())
+        return result
+
+    def push(self, verb):
+        self.verbs.append(verb)
+
+
+class Pause(object):
```
Comment (lfavaro-daitan) [inline bandwidth_sdk/xml.py]: Pause is currently not supported, not sure if we should implement it in the SDK, up to you. For instance we do not have it in the java sdk

Comment (dpereira-daitan) [inline bandwidth_sdk/xml.py]: Good catch, but it is on the acceptance criteria.

File: bandwidth_sdk/xml.py [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+            self._param["locale"] = str(locale)
+        if voice is not None:
+            self._param["voice"] = voice
+
+    # Turn a simple dict of key/value pairs into XML
+    def to_xml(self):
+        return lb.E.SpeakSentence(self._param, self.text)
+
+
+class Gather(object):
```
Comment (lfavaro-daitan) [inline bandwidth_sdk/xml.py]: Missing maxDigits for Gather verb

File: bandwidth_sdk/xml.py [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+    # Turn a simple dict of key/value pairs into XML
+    def to_xml(self):
+        return lb.E.Redirect("", self._param)
+
+
+class Reject(object):
+    """
+        The Reject verb is used to reject incoming calls
+    """
+    @staticmethod
```
Comment (lfavaro-daitan) [inline bandwidth_sdk/xml.py]: Why only this to_xml has @staticmethod annothation?

File: bandwidth_sdk/xml.py [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+    # Turn a simple dict of key/value pairs into XML
+    def to_xml(self):
+        return lb.E.SendMessage(self._param, self.text)
+
+
+class Hangup(object):
+    """
+        The Hangup verb is used to hangup current call
+    """
+    @staticmethod
```
Comment (lfavaro-daitan) [inline bandwidth_sdk/xml.py]: Why use @staticmethod here

File: bandwidth_sdk/xml.py [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+        if "speakSentence" in self._subparam and self._subparam["speakSentence"] is not None:
+            sub_parameters.append(self._subparam["speakSentence"].to_xml())
+        return sub_parameters
+
+    # Turn a simple dict of key/value pairs into XML
+    def to_xml(self):
+        return lb.E.Gather(self._param, *self._get_sub_parameters())
+
+
+class PlayAudio(object):
```
Comment (lfavaro-daitan) [inline bandwidth_sdk/xml.py]: digits is not support by catapult API, but implementation here looks good

File: bandwidth_sdk/xml.py [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+class SendMessage(object):
+    """
+        The SendMessage is used to send a text message
+    """
+    def __init__(self, text, from_number, to_number, status_callback_url=None, request_url=None,
+                 request_url_timeout=None):
+        self._param = OrderedDict()
+        self.text = text
+        if from_number is not None:
+            self._param["from_number"] = str(from_number)
```
Comment (lfavaro-daitan) [inline bandwidth_sdk/xml.py]: The parameter name is incorrect. The correct is:
<xs:attribute name="from"/>
<xs:attribute name="to"/>

**Comment by lfavaro-daitan** [comment]
After applying fixes and fix and pass unit test, please consider Approved. Nice Work!

File: bandwidth_sdk/xml.py [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+    def __init__(self, duration):
+        self._param = OrderedDict()
+        self._param["duration"] = str(duration)
+
+    # Turn a simple dict of key/value pairs into XML
+    def to_xml(self):
+        return lb.E.Pause("", self._param)
+
+
+class Transfer(object):
```
Comment (jsilva-daitan) [inline bandwidth_sdk/xml.py]: Include attributes, recordingEnabled, callTimeout, recordingCallbackUrl, transcribe, etc....

File: bandwidth_sdk/xml.py [hunk 8]
```diff
@@ -0,0 +1,269 @@
...
+        return sub_parameters
+
+    # Turn a simple dict of key/value pairs into XML
+    def to_xml(self):
+        if "url_list" in self._subparam and self._subparam["url_list"] is not None:
+            return lb.E.Media(*self._get_sub_parameters())
+        return lb.E.Media()
+
+
+class Record(object):
```
Comment (jsilva-daitan) [inline bandwidth_sdk/xml.py]: Missing attribute fileFormat

File: bandwidth_sdk/xml.py (hunk 7 above)
Comment (jsilva-daitan) [inline bandwidth_sdk/xml.py]: Missing PlayAudio as inner tag

File: bandwidth_sdk/xml.py (hunk 2 above)
Comment (dpereira-daitan) [inline bandwidth_sdk/xml.py]: included.

File: bandwidth_sdk/xml.py (hunk 3 above)
Comment (dpereira-daitan) [inline bandwidth_sdk/xml.py]: it is a PyCharm recomendation

File: bandwidth_sdk/xml.py (hunk 4 above)
Comment (dpereira-daitan) [inline bandwidth_sdk/xml.py]: it is a PyCharm recomendation

File: bandwidth_sdk/xml.py (hunk 5 above)
Comment (dpereira-daitan) [inline bandwidth_sdk/xml.py]: Note added to the Bxml documentation

File: bandwidth_sdk/xml.py (hunk 6 above)
Comment (dpereira-daitan) [inline bandwidth_sdk/xml.py]: from is a reserved keyword in python. I was just making it consistent

File: bandwidth_sdk/xml.py (hunk 7 above)
Comment (dpereira-daitan) [inline bandwidth_sdk/xml.py]: Added.

File: bandwidth_sdk/xml.py (hunk 8 above)
Comment (dpereira-daitan) [inline bandwidth_sdk/xml.py]: Added.

**Pull request merged**